templates/components/_nav_desktop.html.twig

```diff
@@ -0,0 +1,213 @@
+<nav class="gtm__navigation__desktop l__wrapper l__row l__row--h-stretch l__hide--on-small-laptop {{ app.user ? '' : 'space--20-0' }}">
+    <ul class="list__links list__links--row list__links--no-decor list__links--nav">
+        {% if app.user %}
+            <li>
+                <h1><a href="{{ path('homepage') }}">EM!</a></h1>
+            </li>
+        {% else %}
+            <li>
+                <h1><a href="{{ path('homepage') }}">En Marche !</a></h1>
+            </li>
+            <li>
+                <a href="{{ path('page_le_mouvement') }}">Le Mouvement</a>
+            </li>
+        {% endif %}
+        <li>
+            <a href="{{ path('app_search_events') }}">Événements</a>
+        </li>
+        <li>
+            <a href="{{ path('app_citizen_project_landing') }}">Projets citoyens</a>
+        </li>
+        <li>
+            <a href="{{ path('page_campus') }}">Campus</a>
+        </li>
+        <li>
+            <a href="{{ path('articles_list', { category: 'tout' }) }}">Actualités</a>
+        </li>
+        {% if app.user and not has_role_adherent %}
+            <li>
+                <a href="{{ path('app_membership_register') }}">Devenir adhérent(e)</a>
+            </li>
+        {% endif %}
+        <li>
+            <a href="{{ path('donation_index') }}">Donner</a>
+        </li>
+        <li {% if not algolia_app_id or not algolia_api_key %}style="display: none;"{% endif %}>
+            <a href="javascript:void(0)" class="je-cherche">
+                {{ include('components/search.html.twig') }}
+            </a>
+        </li>
+    </ul>
+    <div class="l__row">
+        {% if not campaign_expired %}
+            {% if is_granted('CREATE_CITIZEN_PROJECT') %}
+                <a class="btn btn--blue" href="{{ path('app_adherent_create_citizen_project') }}">
+                    Lancer mon projet
+                </a>
+            {% endif %}
+            <div class="nav-dropdown">
+                {% if is_granted('ROLE_ADMIN_DASHBOARD') %}
+                    <a href="{{ path('sonata_admin_dashboard') }}" class="nav-dropdown__name">
+                        Admin 🤓
+                    </a>
+                {% else %}
+                    {% if app.user %}
+                        <a href="{{ has_role_adherent ? path('app_user_profile') }}" class="icon--with-text nav-dropdown__name">
+                            {{ include('components/user.html.twig') }}
+                            {{ app.user.firstName is defined ? app.user.firstName : ''}}
+                        </a>
+                    {% endif %}
+                    {% if has_role_user %}
```

Comment:
This if can be merged with above, cannot it?

Reply:
Not the same, if you are admin you don't have this role.